src/main/java/frc/robot/RobotContainer.java

```diff
@@ -64,8 +89,17 @@ public Command getAutonomousCommand() {
    * Adds the test commands to shuffleboard so they can be run that way.
    */
   public void addTestCommands() {
-    ShuffleboardTab tab = Shuffleboard.getTab("Test");
-    tab.add("Do Nothing", new DoNothing());
+
+    GenericEntry testEntry = m_testTab.add("Test Results", false).getEntry();
+    m_testTab.add("Circle Drive", new CircleDrive(m_drive, m_drive.getRequestedDriveVelocityEntry(), m_drive.getRequestedSteerVelocityEntry()));
```

Comment:
Is there a reason that the shuffleboard entries are passsed in even though they can be gotten from the m_drive object that CircleDrive has access to?

Reply:
Same for all of the below commands.

Reply:
no not really in fact the single steer feedforward  gets the module chooser though m_drive. either way it is changed now. ovoius the test enrty how ever is on a diffrent tab and not stored in drive so it is staying.